extensions/packageManager/src/node/index.ts

```diff
@@ -58,6 +119,35 @@ export default async (composer: IExtensionRegistration): Promise<void> => {
         recentlyUsed: recentlyUsed,
       });
     },
+    getFeed: async function (req, res) {
+      // read the list of sources from the config file.
```

Comment:
I think this comment is here by mistake.

Reply:
thx